packages/react-native-reanimated/src/hook/useAnimatedStyle.ts

```diff
@@ -511,11 +514,44 @@ For more, see the docs: \`https://docs.swmansion.com/react-native-reanimated/doc
           areAnimationsActive,
           isAnimatedProps
         );
+        if (
+          Platform.OS === 'android' &&
+          !globalThis._IS_FABRIC &&
+          remoteState.isFirstRun
+        ) {
+          /* 
+            This is a makeshift patch for a bug on Paper Android.
+
+            When an Animated Component gets mounted and it already has an
+            update (i.e. animation starts on mount), the first update can be 
+            lost. This is due to fact that Android UIManager is batching the
+            Native Views creation - an update might come before such batch
+            is executed, in result trying to modify a view which doesn't exist.
+            
+            After failing to force the UIManager 
+            to create the View pre-update, I decided to instead schedule the 
+            first update twice for Animated Style, because on another
+            frame the view should exist and the update should be successful.
+          */
+          requestAnimationFrame(() =>
+            styleUpdater(
+              shareableViewDescriptors,
+              updater,
+              remoteState,
+              areAnimationsActive,
+              isAnimatedProps
+            )
+          );
```

Comment:
We should consider resolving this in native code - similar to the fixes we've already implemented for iOS:
- https://github.com/software-mansion/react-native-reanimated/pull/2580
- https://github.com/software-mansion/react-native-reanimated/pull/5224

The current solution seems a bit undeterministic and could potentially cause some unexpected blinking 🤔

Reply:
Honestly I don't believe it would lead to blinks, since the update is never stale - but probably during some heavy traffic and slow JS/UI the update could still get lost (although a lot less likely).

Implementing it in the native code is the best solution here, but then it's a code that would need to be maintained and it steers us away from improving Reanimated for Fabric 😿